tests/functional/test_show.py

```diff
@@ -151,6 +152,7 @@ def test_pip_show_is_short(script):
     assert len(lines) <= 10
 
 
+@pytest.mark.network
```

Comment:
I've added it in `test_package_name_is_canonicalized`. Should I fix it for the `test_pip_show_divider` too?